packages/eslint-plugin-twenty/dist/index.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
Probably not intended to push all of this?

Reply:
👍